ide/xsl/src/org/netbeans/modules/xsl/grammar/XSLGrammarQuery.java

```diff
@@ -136,33 +136,33 @@ private static Map getElementDecls() {
             // Commonly used variables
             Set emptySet = new TreeSet();
             String spaceAtt = "xml:space";  // NOI18N
-            Set tmpSet;
+            Set<String> tmpSet;
 
             ////////////////////////////////////////////////
             // Initialize common sets
 
-            Set charInstructions = new TreeSet(Arrays.asList(new String[]{"apply-templates", // NOI18N
+            Set<String> charInstructions = new TreeSet<>(Arrays.<String>asList(new String[]{"apply-templates", // NOI18N
```

Comment:
inferrable type informations
```suggestion
            Set<String> charInstructions = new TreeSet<>(Arrays.asList(new String[]{"apply-templates", // NOI18N
```
strictly speaking the array creation is not needed as well

Reply:
changed..

Reply:
changed..